tests/integration/util/features.rs

```diff
@@ -19,6 +20,7 @@ impl From<&str> for TestFeature {
             "queryindex" => TestFeature::QueryIndex,
             "queryindexdefs" => TestFeature::QueryIndexDefinitions,
             "queryindexadvise" => TestFeature::QueryIndexAdvise,
+            "capella" => TestFeature::Capella,
```

Comment:
I'm unsure that this is the correct way to approach this. Couldn't we use the presence of the access/secret keys to indicate capella support?